cli/tests/041_info_flag_location.out

```diff
@@ -0,0 +1,2 @@
+Origin data: "[WILDCARD]location_data[WILDCARD]"
```

Comment:
I keep interpreting this as "origin info" as opposed to "per-origin storage bucket". How about:
```suggestion
Origin storage: "[WILDCARD]location_data"
```

Reply:
Done